requirements.txt

```diff
@@ -1,4 +1,4 @@
-codespell==1.14.0
+codespell==1.15.0
 cpplint==1.4.4
-awscli==1.16.135 
+awscli==1.16.16
```

Comment:
Typo? https://pypi.org/project/awscli/ says the latest version is `1.16.161`

Reply:
I used github again.

Reply:
I'll change it.

Reply:
Fixed.

Reply:
https://pypi.org/project/awscli/ now says `awscli 1.16.176` :wink:

Reply:
ok